.github/workflows/starknet-js-tests.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         run: npm ci
   
       - name: Run tests
-        run: npm test
+        run: npm test -- rpcProvider.test.ts transactionReceipt.test.ts rpcChannel.test.ts defaultProvider.test.ts contract.test.ts cairo1v2.test.ts cairo1v2_typed.test.ts cairo1.test.ts account.test.ts account.starknetId.test.ts --testNamePattern="^(?!.*traceTransaction|.*declare Sierra 1.5.0).*$"
```

Comment:
It appears that starknet.js includes tests beyond the rpc provider scope, hence selecting specific test files for running. Excluded `traceTransaction` for now due to a reported Juno bug and skipped `declare Sierra 1.5.0` since it's meant for devnet - it fails on live networks as contracts with the same hash can't be declared twice.